study-builder/studies/testboston/adhoc-symptom.conf

```diff
@@ -0,0 +1,50 @@
+{
+  include required("../snippets/activity-general-form.conf"),
+  "studyGuid": ${id.study},
+  "activityCode": ${id.act.adhoc_symptom},
+  "versionTag": "v1",
+  "displayOrder": 5,
+  "writeOnce": false,
+  "maxInstancesPerUser": 1,
+  "translatedNames": [
+    { "language": "en", "text": ${i18n.en.adhoc_symptom.name} },
+    { "language": "es", "text": ${i18n.es.adhoc_symptom.name} },
+    { "language": "ht", "text": ${i18n.ht.adhoc_symptom.name} }
+  ],
+  "translatedTitles": [
+    { "language": "en", "text": ${i18n.en.adhoc_symptom.title} },
+    { "language": "es", "text": ${i18n.es.adhoc_symptom.title} },
+    { "language": "ht", "text": ${i18n.ht.adhoc_symptom.title} }
+  ],
+  "translatedSummaries": [
```

Comment:
activity needs summary text in dashboard rather than testboston standard question numbers/status code.